glotaran/project/result.py

```diff
@@ -170,47 +262,70 @@ def markdown(self, with_model: bool = True, base_heading_level: int = 1) -> Mark
         return MarkdownStr(result_table)
 
     def _repr_markdown_(self) -> str:
-        """Special method used by ``ipython`` to render markdown."""
+        """Return a markdown representation str.
+
+        Special method used by ``ipython`` to render markdown.
+
+        Returns
+        -------
+        str
+            The scheme as markdown string.
+        """
         return str(self.markdown(base_heading_level=3))
 
     def __str__(self):
+        """Overwrite of ``__str__``."""
         return str(self.markdown(with_model=False))
 
-    @deprecate(
-        deprecated_qual_name_usage="glotaran.project.result.Result.save(result_path)",
-        new_qual_name_usage=(
-            "glotaran.io.save_result("
-            "result=result, result_path=result_path, "
-            'format_name="legacy", allow_overwrite=True'
-            ")"
-        ),
-        to_be_removed_in_version="0.6.0",
-        importable_indices=(2, 1),
-    )
     def save(self, path: str) -> list[str]:
-        """Saves the result to given folder.
+        """Save the result to given folder.
 
-        Warning
-        -------
-        Deprecated use ``save_result(result_path=result_path, result=result,
-        format_name="legacy", allow_overwrite=True)`` instead.
+        Parameters
+        ----------
+        path : str
+            The path to the folder in which to save the result.
 
+        Returns
+        -------
+        list[str]
+            Paths to all the saved files.
+        """
+        return cast(
+            List[str],
+            save_result(result_path=path, result=self, format_name="folder", allow_overwrite=True),
+        )
 
-        Returns a list with paths of all saved items.
-        The following files are saved:
+    def recreate(self) -> Result:
```

Comment:
My first thought when seeing recreate is, does this do the optimization or not, so that should be added to the docstring. And there may be value in providing both "recreate" as in construct_and_optimize and "constuct" as in "construct_but_do_not_optimize". 

Also fix typos in docstring.

Reply:
Well I thought "recreate" implies that it is optimized again from initial parameters. But I would leave that for follow up PR because I am sure you have more requirements for this this feature.